src/Tonic/Noise.cpp

```diff
@@ -14,9 +14,9 @@
 namespace Tonic {
   namespace Tonic_ {
   
-    PinkNoise_::PinkNoise_() : pinkCount_(1), pinkAccum_(0) {
+    PinkNoise_::PinkNoise_() :pinkAccum_(0),  pinkCount_(1) {
```

Comment:
Some compilers issue warnings when the initalizer lists are not in the same order as the class members. This is because in C++ members are always initialized in the order they are declared, regardless of the order they appear in initializer lists.